docs/references/general/grid.md

```diff
@@ -3,44 +3,30 @@ author(s): Marina Schmidt
 
 Disclaimer: License under GNU v3.0, the file is found in the root directory under LICENSE 
 
+Modifications:
+    2022-11-28 Emma Bland (UNIS) Updated file format description
+
 -->
 # GRID files 
 
-GRID files are post-processed data produced from FITACF files.
-
-## Naming Conventions
-
-### Single radar GRID files
-
-Currently the common naming convention for a single radar GRID file is:
-
-> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.grid
+GRID files are post-processed data produced from FITACF files, in which the fitted data products are placed on a magnetic latitude/longitude grid of equal-area cells spanning 1 degree of magnetic latitude. They may contain data from one radar or from multiple radars. For more information, see the [`make_grid` tutorial](../../user_guide/make_grid.md).
 
-Some radars provide separate data files for each channel. In this case, the channel is specified after the 3-letter station ID:
+## Naming Convention
 
-> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.[a-d].grid
+The community standard for naming GRID files containing data from only one radar is:
 
-See [fitacf](fitacf.md) documentation for more information on this naming convention. 
+> YYYYMMDD.<3-letter abbreviation>.grd
 
-If a GRID file is produced for 24-hours using the `-c` option in `make_grid` (see [`make_grid` tutorial](../../user_guide/make_grid.md)) or by combining individual GRID files with `combine_grid`, then the common naming convention is:
+When the gridded data from multiple radars have been combined, the files are named as:
```

Comment:
Could/should there be a quick statement that grid files with multiple radars are created with radars from the same hemisphere?  This is an assumption that I don't think I've ever thought about, but is one that is made with the gridding processing.